StudCity.Frontend/src/services/googleService.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+export const googleAuthenticate = async (token) => {
+    try {
+        const { data } = await axios.get(`https://www.googleapis.com/oauth2/v3/userinfo?access_token=${token}`);
+        if (data) {
```

Comment:
is it possible for api endpoint on line 5 return non-truthy value? if yes then okay, if not then this if statement unnecessary

Reply:
I think everything is possible in our profession) So therefore, this check will not be superfluous